### PR TITLE
wrote a css for index.html

### PR DESCRIPTION
**Pull request by Vanderwise**
Please check my css for fierst page. There are a lot of line of code so maybe it would be better to divide comments by topics into blocks and work gradually with it.
You can offer your way of working, you are the master here ;) But I would start from BEM, becouse I feel that i'm not really good in it and becouse of it all css markdown looks like mess. 
It's not full css and many points are still missing (some :hover and :focus) so if you see a obvious incompleteness just know that i will add it in the next pull request.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -258,4 +548,118 @@ p.content-description {
...
+    margin-top: auto;
+    margin-bottom: auto;
+    background-color: #ffc600;
+}
+.socials-links {
+    width: 160px;
+}
+.socials-links a {
+    opacity: 30%;
 }
```
Comment (ratetenyov) [inline css/style.css]: no end line

File: index.html [hunk 2]
```diff
@@ -8,34 +8,42 @@
...
                 <a href=".//index.html"><img src=".//img/logo-black.png" alt="Логотип надпись device черным цветом"></a>
-            </span>
+            </div>
             <div class="header-buttons">                
-                <div class="search">                    
+                <div class="search-container">                    
                     <label class="visually-hidden" for="input-search">Поиск по сайту</label>
                     <input class="input-search" type="text" name="search" id="input-search" placeholder="Поиск по сайту">                                    
-                    <span class="button-search"><a href="#">Найти</a></span>
+                    <div class="button-search"><a href="#">Найти</a></div>
```
Comment (ratetenyov) [inline index.html]: interactive element, better use button

File: index.html [hunk 3]
```diff
@@ -8,34 +8,42 @@
...
                 </div>
                 <div class="user-buttons">
                     <ul class="user-registration">
-                        <li class="button-registration"><a href=".//registration.html">Войти</a></li>
+                        <li class="button-registration" tabindex="-1"><a href=".//registration.html">Войти</a></li>
                     </ul>
                     <ul class="user-purchase">
-                        <li class="button-compare"><a href="#">Сравнить</a></li>
+                        <li class="button-compare" tabindex="-1"><a href="#">Сравнить</a></li>
```
Comment (ratetenyov) [inline index.html]: why tabindex -1, what for do you remove this link from navigation flow?

File: index.html [hunk 4]
```diff
@@ -8,34 +8,42 @@
...
-                <ul class="nav-list">
-                    <li class="nav-catalog"><a href="#">Каталог товаров</a></li>
-                    <li class="#"><a href="#">Доставка</a></li>
-                    <li class="#"><a href="#">Гарантия</a></li>
-                    <li class="#"><a href="#">Контакты</a></li>
-                </ul>
+            <nav class="nav" aria-label="Навигация по сайту">
+                <div class="div-catalog">
+                    <ul class="ul-catalog">
+                        <li class="li-catalog"><a href="#">Каталог товаров</a></li>
```
Comment (ratetenyov) [inline index.html]: where are catalog items, user sees on hover?

File: index.html [hunk 5]
```diff
@@ -8,34 +8,42 @@
...
-            <nav class="nav" aria-label="Навигация по сайту">                                                
-                <ul class="nav-list">
-                    <li class="nav-catalog"><a href="#">Каталог товаров</a></li>
-                    <li class="#"><a href="#">Доставка</a></li>
-                    <li class="#"><a href="#">Гарантия</a></li>
-                    <li class="#"><a href="#">Контакты</a></li>
-                </ul>
+            <nav class="nav" aria-label="Навигация по сайту">
+                <div class="div-catalog">
+                    <ul class="ul-catalog">
```
Comment (ratetenyov) [inline index.html]: list with one item... is it a list at all? I would use only link with such layout. In my project, for ex, Каталог, Доставка Гарантия, Контакты are all in one `ul`. Doesn't mean you should do the same, but ul with one item is not an ul, I guess)

File: index.html [hunk 6]
```diff
@@ -8,34 +8,42 @@
...
+                    </ul>
+                </div>    
+                <div class="div-services">
+                    <ul class="ul-services">
+                        <li class="#"><a href="#">Доставка</a></li>
+                        <li class="#"><a href="#">Гарантия</a></li>
+                    </ul>
+                    <ul class="ul-contacts">
+                        <li class="#"><a href="#">Контакты</a></li>
+                    </ul>
```
Comment (ratetenyov) [inline index.html]: too many `uls` here! try to combine these two in one.

File: index.html [hunk 7]
```diff
@@ -48,22 +56,24 @@
...
+        <main class="main-container">
             <h1 class="visually-hidden">Интернет магазин device</h1>            
-            <section class="slider">
+            <section class="section-slider">
                 <h2 class="visually-hidden">Реклама промо товаров</h2>
-                <div class="slider">
+                <div class="slider-container">
                     <div class="slider-item">
                         <div class="slider-img">
                             <img src=".//img/content-img/slider-1.png" alt="Три телескопические палки для селфи с мягкой рукояткой. Основной цвет - черный, декоративный цвет - синий">
```
Comment (ratetenyov) [inline index.html]: too long alt)

Comment (ratetenyov) [inline index.html]: what for div wrapper for img? do you really need it?

File: index.html [hunk 8]
```diff
@@ -48,22 +56,24 @@
...
                         </div>
                         <div class="slider-content">
-                            <!-- <p>01</p> Псевдоэлемент-->
+                            <p class="slider-content-number">01</p>
                             <p class="content-tagline">Делай селфи, <br>как Бен Стиллер!</p>
                             <p class="content-description">Самая длинная палка для селфи доступна в нашем магазине. <br>Восемь (Восемь Карл!) метров и весом всего 5 килограмм.</p>
-                            <a class="content-link" href="#">подробнее</a>
+                            <div class="content-button">
```
Comment (ratetenyov) [inline index.html]: really need this div?

File: index.html [hunk 9]
```diff
@@ -85,10 +95,12 @@ <h2 class="visually-hidden">Реклама промо товаров</h2>
                             <img src=".//img/content-img/slider-2.png" alt="фитнес-браслет черного цвета, резиновый корпус, поверхность ребристая">
                         </div>
                         <div class="slider-content">
-                            <p>02</p>
+                            <!-- <p class="slider-content-number">02</p> -->
                             <p class="content-tagline">Худеем <br>правильно!</p>
                             <p class="content-description">Мотивирующие фитнес-браслеты помогут найти в себе силы <br>не пропускать занятия и соблюдать диету.</p>
-                            <a class="content-link" href="#">подробнее</a>
+                            <div class="content-button">
```
Comment (ratetenyov) [inline index.html]: div?

File: index.html [hunk 10]
```diff
@@ -106,10 +118,12 @@ <h2 class="visually-hidden">Реклама промо товаров</h2>
                             <img src=".//img/content-img/slider-3.png" alt="квадрокоптер белого цвета с лазерной пушкой вместо камеры">
                         </div>
                         <div class="slider-content">
-                            <p>03</p>
+                            <!-- <p class="slider-content-number">03</p> -->
                             <p class="content-tagline">Пархает как бабочка, <br>жалит как пчела!</p>
                             <p class="content-description">Этот обычный, на первый взгляд, квадрокоптер оснащен <br>мощным лазером, замаскированным под стандартную камеру.</p>
-                            <a class="content-link" href="#">подробнее</a>
+                            <div class="content-button">
```
Comment (ratetenyov) [inline index.html]: and here

File: index.html [hunk 11]
```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
...
-                    <li>ЖелДорЭкспедиция</li>
-                </ul>
-                <a href="#">Подробнее о нас</a>
-            </section>
-            <section class="contacts">
-                <h2>Контакты</h2>
-                <p>Вы можете забрать товар сами, заехав в наш офис. Заодно, вы сможете проверить работоспособность покупки. Всякое бывает.</p>                
-                <div>
-                    <img src=".//img/map.jpg" width="560" height="222" alt="Наш магазин расположен по адресу г. Москва, ул. Строителей, д.15">
+                <div class="logotypes">
```
Comment (ratetenyov) [inline index.html]: why this div?

File: index.html [hunk 12]
```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
...
-                        <div class="div_text">
-                            <label for="text">Текст письма:</label>
-                            <textarea id="text" name="user-text" placeholder="В свободной форме"></textarea>
-                        </div>
-                        <button type="submit">Отправить</button>                            
-                    </form>
-                </div>                
-            </section>                    
+            </section>    
+            <section class="section-info">
```
Comment (ratetenyov) [inline index.html:193]: all section must have headers, at least hidden. think of just div wrapper for these two sections

File: index.html [hunk 13]
```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
...
+                <section class="about-us">
+                    <h2>О нас</h2>
+                    <p>Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</p>
+                    <p>Мы можем доставить ваш товар в самые отдаленные точки России! DEVICE работает со многими транспортными компаниями</p>
```
Comment (ratetenyov) [inline index.html]: missing colon after компаниями)

File: index.html [hunk 14]
```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
...
+                    <ul class="delivery-list">
+                        <li>Деловые линии</li>
+                        <li>Автотрейдинг</li>
+                        <li>ЖелДорЭкспедиция</li>
+                    </ul>
+                    <div class="button">
```
Comment (ratetenyov) [inline index.html]: div?

File: index.html [hunk 15]
```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
...
+                    
+                </section>
+                <section class="contacts">
+                    <h2>Контакты</h2>
+                    <p>Вы можете забрать товар сами, заехав в наш офис. Заодно, вы сможете проверить работоспособность покупки. Всякое бывает.</p>                
+                    <div class="contacts-map">
+                        <img src=".//img/map.jpg" width="560" height="222" alt="Наш магазин расположен по адресу г. Москва, ул. Строителей, д.15">
+                    </div>
+    <!-- смотри раздел 4.3 specification-device, доделай, когда изучишь соответствующую тему  -->
+                    <div class="button">
```
Comment (ratetenyov) [inline index.html]: div!

File: index.html [hunk 16]
```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
...
+                        <button class="contacts-button" type="button">Напишите нам</button>
```
Comment (ratetenyov) [inline index.html]: link

File: index.html [hunk 17]
```diff
@@ -8,34 +8,42 @@
     </head>
     <body>
         <header class="header">
-            <span class="logo-link">
+            <div class="logo-link">
```
Comment (ratetenyov) [inline index.html]: by the way, on the main page logo doesn't lead anywhere

File: css/style.css [hunk 18]
```diff
@@ -51,105 +64,243 @@ ul {
...
     font-size: 15px;
 }
-.logo-link {
-    margin: 23px 0 0 199px;
-    width: 100%;
-    height: 100%;
+.user-buttons {
+    display: flex;
+    justify-content: space-between;
+    width: 500px;
```
Comment (ratetenyov) [inline css/style.css]: I think this will make your UI not responsive

File: css/style.css [hunk 19]
```diff
@@ -51,105 +64,243 @@ ul {
...
+}
+.search-container {
+    display: flex;
+    position: relative;
+    box-sizing: border-box;
+    align-items: center;
+    width: 500px;
```
Comment (ratetenyov) [inline css/style.css]: same

File: css/style.css [hunk 20]
```diff
@@ -51,105 +64,243 @@ ul {
...
-    height: 100%;
+.user-buttons {
+    display: flex;
+    justify-content: space-between;
+    width: 500px;
+}
+.search-container {
+    display: flex;
+    position: relative;
+    box-sizing: border-box;
```
Comment (ratetenyov) [inline css/style.css]: already in global styles

File: css/style.css [hunk 21]
```diff
@@ -51,105 +64,243 @@ ul {
...
+    top: 17px;
+    z-index: 10;
 }
-.input-search {    
+.input-search {  
+    height: 49px; 
+    padding-left: 36px; 
     opacity: 30%;
     border: none;
+    box-sizing: border-box;
```
Comment (ratetenyov) [inline css/style.css]: ➖

File: css/style.css [hunk 22]
```diff
@@ -51,105 +64,243 @@ ul {
...
-    content: url(.//img/search.svg);    
-    padding: 0 19px 0 0;
+.button-search {
+    display: none;
+}
+.input-search:focus + div {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
```
Comment (ratetenyov) [inline css/style.css]: delete it from everywhere please

File: css/style.css [hunk 23]
```diff
@@ -51,105 +64,243 @@ ul {
...
+    width: 85px;
+    height: 49px;
+    border: 2px solid #000000;
+}
+.user-buttons ul {
+    padding: 0;
```
Comment (ratetenyov) [inline css/style.css]: have it in global

File: css/style.css [hunk 24]
```diff
@@ -51,105 +64,243 @@ ul {
...
     display: flex;
-    justify-content: center;
-    padding: 0 20px;
+    align-items: center;
+    justify-content: space-between;
+    width: auto;
+    padding-bottom: 58px;
+    margin-left: 60px;
+    margin-right: 60px;
+    font-size: 18px;
```
Comment (ratetenyov) [inline css/style.css:226]: bad practice to specify font params here, do it in the selector, related to content of nav or ul, in your case it's probably link

File: css/style.css [hunk 25]
```diff
@@ -51,105 +64,243 @@ ul {
...
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
 }
-.user-purchase li {
-    padding: 0 0 0 17px;
+.div-catalog {
+    display: flex;
+    justify-content: left;
+    width: 500px;
```
Comment (ratetenyov) [inline css/style.css]: all constant widths make your site not responsive. almost all

File: css/style.css [hunk 26]
```diff
@@ -51,105 +64,243 @@ ul {
...
+    margin: 0 140px 0 140px;     */
     font-family: "Opensanslight", sans-serif;
     font-size: 15px;
     line-height: 36px;
-    background-color: #f7e184;
+    background-color: #f7e296;
+}
+.section-slider {
+    max-width: 1160px;
+    height: auto;
```
Comment (ratetenyov) [inline css/style.css]: really need it?

File: index.html [hunk 27]
```diff
@@ -48,22 +56,24 @@
...
                 <h2 class="visually-hidden">Реклама промо товаров</h2>
-                <div class="slider">
+                <div class="slider-container">
                     <div class="slider-item">
                         <div class="slider-img">
                             <img src=".//img/content-img/slider-1.png" alt="Три телескопические палки для селфи с мягкой рукояткой. Основной цвет - черный, декоративный цвет - синий">
                         </div>
                         <div class="slider-content">
-                            <!-- <p>01</p> Псевдоэлемент-->
+                            <p class="slider-content-number">01</p>
```
Comment (ratetenyov) [inline index.html:55]: is has position absolute in styles. if so, better make it as pseudo element

File: css/style.css [hunk 28]
```diff
@@ -51,105 +64,243 @@ ul {
...
     font-family: "Opensanslight", sans-serif;
     font-size: 15px;
     line-height: 30px;
 }
+.content-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 220px;
+    height: 40px;
```
Comment (ratetenyov) [inline css/style.css]: don't use fixed height for such elements. your height shoul be defined only by font-size, line-height, and paddings

File: css/style.css [hunk 29]
```diff
@@ -159,6 +310,25 @@ p.content-description {
     height: 40px;    
     color: rgba(0, 0, 0, 0.3);
 }
+.content-properties tr {
+    display: flex;
+    justify-content: left;
+}
+.content-properties th:nth-child(1) {
```
Comment (ratetenyov) [inline css/style.css]: please enumerate common properties for these children through comma

File: css/style.css [hunk 30]
```diff
@@ -170,17 +340,31 @@ p.content-description {
     font-size: 13px;
     font-weight: 400;
     line-height: 20px;
+} 
+.section-popular {
+    max-width: 1160px;
+    height: auto;
```
Comment (ratetenyov) [inline css/style.css]: what for do you use this auto everywhere?

File: css/style.css [hunk 31]
```diff
@@ -170,17 +340,31 @@ p.content-description {
...
+.popular-content {
+    display: flex;
+    align-items: flex-start;
+    width: 1160px;
 }
 .popular-content-list {
     display: flex;
     align-items: flex-start;
+    justify-content: space-between;
+    width: 1160px;
```
Comment (ratetenyov) [inline css/style.css:360]: bad

File: css/style.css [hunk 32]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+.services-container {
     display: flex;
+    justify-content: flex-start;
     align-items: center;
+    width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    height: auto;
+}
+.services::before {
```
Comment (ratetenyov) [inline css/style.css]: this can be just background with linear-gradient

File: css/style.css [hunk 33]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+    z-index: -1;
 }
+.services-button {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    min-height: 180px;
+    min-width: 280px;
+    font-size: 18px;
```
Comment (ratetenyov) [inline css/style.css:472]: it's better to use font-family, and all set of font props in the elements with text content, like links or paragraphs of buttons

File: css/style.css [hunk 34]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+    line-height: 24px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+}
+.btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 160px;
+    height: 40px;
```
Comment (ratetenyov) [inline css/style.css]: bad

File: css/style.css [hunk 35]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+    font-size: 18px;
+    line-height: 24px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+}
+.btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 160px;
```
Comment (ratetenyov) [inline css/style.css]: bad

File: css/style.css [hunk 36]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+    height: 40px;
+    box-sizing: border-box;
```
Comment (ratetenyov) [inline css/style.css]: remove it from everywhere please

File: css/style.css [hunk 37]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+    margin-top: auto;
+    margin-bottom: auto;
+    /* background-color: #f0c52e;  */
+     
+}
+
+.btn:hover {
+    width: 160px;
+    height: 40px;
```
Comment (ratetenyov) [inline css/style.css]: no need width and height

File: css/style.css [hunk 38]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+    background-color: #000000;
+    color: #f0c52e;  
+}
+.delivery-switch~.service-delivery,
```
Comment (ratetenyov) [inline css/style.css]: spaces around tilda please

File: css/style.css [hunk 39]
```diff
@@ -194,10 +378,90 @@ p.content-description {
...
+.warranty-switch~.service-warranty,
+.credit-switch~.service-credit {
+    display: none;
+}
```
Comment (ratetenyov) [inline css/style.css:496]: for the future - separate style rules with empty line

**Comment by ratetenyov** [comment]
main nitpicks: fixed sizes of containers and such elements as buttons and links

File: css/style.css (hunk 1 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: index.html (hunk 5 above)
Comment (Vanderwise) [reply index.html]: Done. DIVATOZ attacted me.

File: index.html (hunk 6 above)
Comment (Vanderwise) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (Vanderwise) [reply index.html]: Removed it. It was after lecture of accessibility and i checked how it works and haven't removed

File: index.html (hunk 2 above)
Comment (Vanderwise) [reply index.html]: Done

File: index.html (hunk 7 above)
Comment (Vanderwise) [reply index.html]: Makeev says that alt have to be explicit. Didn't change

Comment (Vanderwise) [reply index.html]: Removed div, corrected css

File: index.html (hunk 8 above)
Comment (Vanderwise) [reply index.html]: Done. linear-gradient works :)

File: index.html (hunk 9 above)
Comment (Vanderwise) [reply index.html]: Done

File: index.html (hunk 10 above)
Comment (Vanderwise) [reply index.html]: Done

File: index.html (hunk 11 above)
Comment (Vanderwise) [reply index.html]: Removed

File: index.html (hunk 12 above)
Comment (Vanderwise) [reply index.html:193]: if all section must have header then I missed at least two: section class="section-popular" and section class="services" Here have changed section to div. What about these two? Add h?

File: index.html (hunk 13 above)
Comment (Vanderwise) [reply index.html]: Done

File: index.html (hunk 14 above)
Comment (Vanderwise) [reply index.html]: Yep. Removed

File: index.html (hunk 15 above)
Comment (Vanderwise) [reply index.html]: Yep. Removed

File: index.html (hunk 16 above)
Comment (Vanderwise) [reply index.html]: is the modal window summoned by link?

File: index.html (hunk 17 above)
Comment (Vanderwise) [reply index.html]: Agree, done

File: css/style.css (hunk 18 above)
Comment (Vanderwise) [reply css/style.css]: let's discuss it when we call

File: css/style.css (hunk 20 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 21 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 22 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 23 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 24 above)
Comment (Vanderwise) [reply css/style.css:226]: What about li that contains link?

File: css/style.css (hunk 25 above)
Comment (Vanderwise) [reply css/style.css]: it's a big topic to discuss when we call

File: css/style.css (hunk 26 above)
Comment (Vanderwise) [reply css/style.css]: No

File: css/style.css (hunk 29 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 30 above)
Comment (Vanderwise) [reply css/style.css]: Removed.

File: css/style.css (hunk 32 above)
Comment (Vanderwise) [reply css/style.css]: Yep. It can be. Done but now need to work with positioning

File: css/style.css (hunk 33 above)
Comment (Vanderwise) [reply css/style.css:472]: OK, got it

File: css/style.css (hunk 36 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 37 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 38 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: css/style.css (hunk 39 above)
Comment (Vanderwise) [reply css/style.css:496]: OK

File: css/style.css (hunk 18 above)
Comment (Vanderwise) [reply css/style.css]: changed to 50% and margin-left

File: css/style.css (hunk 19 above)
Comment (Vanderwise) [reply css/style.css]: changed to 50% and margin-left

File: css/style.css (hunk 34 above)
Comment (Vanderwise) [reply css/style.css]: Done. have used paddings instead of height

File: css/style.css (hunk 35 above)
Comment (Vanderwise) [reply css/style.css]: Done

File: index.html (hunk 4 above)
Comment (Vanderwise) [reply index.html]: I don't understand how to insert it into html markdown and css

File: index.html (hunk 16 above)
Comment (ratetenyov) [reply index.html]: yes, link

**Review by ratetenyov — APPROVED** [review]
Good job! Getting better with every next PR!

File: index.html (hunk 16 above)
Comment (Vanderwise) [reply index.html]: changed to the link

**Pull request merged**